src/FetchAuthTokenInterface.php

```diff
@@ -26,6 +26,7 @@ interface FetchAuthTokenInterface
      * Fetches the auth tokens based on the current state.
      *
      * @param callable|null $httpHandler callback which delivers psr7 request
+     *
```

Comment:
Nit, Is the @param and @return docs separated by a new line?

Reply:
Using regex, I found 8 instances where we had a newline between `@param` and `@return`, and 92 occurrences where we didn't.

Reply:
```suggestion
```